atmo/appsource/bundlesource.go

```diff
@@ -31,7 +32,14 @@ func NewBundleSource(path string) AppSource {
 	return b
 }
 
-// Start initializes the app source
+// checkIdentifierVersion checks whether the passed in identifier and version are for the current app running in the
+// bundle or not. Returns true only if both match.
+func (b *BundleSource) checkIdentifierVersion(identifier, version string) bool {
```

Comment:
unexported helper functions like this should go towards the bottom of the file

Reply:
Fixed in https://github.com/suborbital/atmo/pull/121/commits/26e7d81844c907f81c3e0f3acc5d6bb468d74635